JASP-Engine/JASP/R/auditClassicalBenfordsLaw.R

```diff
@@ -182,14 +186,20 @@ auditClassicalBenfordsLaw <- function(jaspResults, dataset, options, ...){
       leadingDigits <- table(as.numeric(substring(format(abs(obs), scientific = TRUE), 1, 3)) * 10)
       digits <- 10:99
 
+    } else if(options[["digits"]] == "last"){
+
+      stringedObs <- format(abs(obs), scientific = TRUE)
+      leadingDigits <- table(as.numeric(substring(stringedObs, nchar(stringedObs), nchar(stringedObs))))
+      digits <- 1:9
```

Comment:
Aren't you missing 0?

Reply:
As in, something can end in "0.12300231" right?

Reply:
Yes, but 0 doesn't have a probability according to Benford's Law.

Reply:
Have you tried this with values such as 5.60? What should the procedure do then? Skip 0 the zero and check 6?